app/validators/subject_changed_validator.rb

```diff
@@ -0,0 +1,8 @@
+class SubjectChangedValidator < ActiveModel::EachValidator
```

Comment:
Add the same comments @gabrielbaldao made in his PRs, to remember us to rename it back when removing the gem...